multiple-search-services/README.md

```diff
@@ -53,6 +53,12 @@ Open the **appsettings.json** file in your local copy of the sample application
 
     + Find the URI in the [search service's Overview portal page](https://docs.microsoft.com/azure/search/search-manage#overview-home-page).
 
+1. "semanticConfigurationName": "Semantic Configuration Name on your index":
+
+    + Required if you want to request a Semantic query. Find the Semantic Config in the [search service's desired index in the search explorer under the Configurations tab](https://learn.microsoft.com/en-us/azure/search/semantic-how-to-query-request?tabs=portal%2Cportal-query).
```

Comment:
line 58 -- how about "Required if you want to request semantic reranking. You can choose an existing configuration or [create a new one in the portal](https://learn.microsoft.com/azure/search/semantic-how-to-query-request?tabs=portal%2Cportal-query)".  
Note the casing.
In the link, please remove "en-us" (it hard codes the english version of the article).